backend/experiment/tests/test_admin_experiment.py

```diff
@@ -66,4 +66,4 @@ def test_admin_export(self):
         experiment = Experiment.objects.first()
         response = this_experiment_admin.export(request, experiment)
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response['content-type'], 'application/json')
+        self.assertEqual(response['content-type'], 'application/x-zip-compressed')
```

Comment:
I find this test way too limited. I would want to know that for a toy experiment with several sessions, including linked results, sections and participants, we get the relevant data.